src/App.tsx

```diff
@@ -1,16 +1,21 @@
+import React from 'react';
+
+import { ProductsComponent } from '@/components/products/Products.component.tsx';
+import { AppContextProvider } from '@/context/App.context.tsx';
+
 import { AboutComponent } from './components/about/About.component.tsx';
 import { FooterComponent } from './components/footer/Footer.component.tsx';
 import { HeaderComponent } from './components/header/Header.component.tsx';
 
 function App() {
+    const [activeMenuItem, setActiveMenuItem] = React.useState<string>('products');
```

Comment:
Would be good to add an enum and use its values despite just strings. This will save you from a typos
```suggestion
    const [activeMenuItem, setActiveMenuItem] = React.useState<MenuItem>(MenuItem.PRODUCTS);
```

Reply:
Good advice, thank you. Fixed